www-apps/bugzilla/bugzilla-5.0.6.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
EAPI=7 if we can, but you know this from IRC.

Reply:
Done.